pkg/preflights/host-preflight.yaml

```
@@ -367,17 +371,30 @@ spec:
         regex: 'nameserver\s*(localhost|127\.0\.0\.1)'
         outcomes:
           - fail:
-              when: "true"
+              when: 'true'
               message: "Neither 'nameserver localhost' nor 'nameserver 127.0.0.1' can be present in resolv.conf. Remove them to continue."
```

Makes me wonder if we should reword this one a bit to match more closely. If this wording looks right.
```suggestion
              message: "Local DNS resolver detected. Remove the localhost and/or 127.0.0.1 nameserver entries from resolv.conf."
```